xml/System.Linq/IQueryable.xml

```diff
@@ -44,7 +44,7 @@
   
  The <xref:System.Linq.IQueryable> interface inherits the <xref:System.Collections.IEnumerable> interface so that if it represents a query, the results of that query can be enumerated. Enumeration causes the expression tree associated with an <xref:System.Linq.IQueryable> object to be executed. The definition of "executing an expression tree" is specific to a query provider. For example, it may involve translating the expression tree to an appropriate query language for the underlying data source. Queries that do not return enumerable results are executed when the <xref:System.Linq.IQueryProvider.Execute%2A> method is called.  
   
- For more information about how to create your own LINQ provider, see [LINQ: Building an IQueryable Provider](https://go.microsoft.com/fwlink/?LinkID=112370) on MSDN Blogs.  
+ For more information about how to create your own LINQ provider, see [LINQ: Building an IQueryable Provider](https://github.com/mattwar/iqtoolkit/tree/master/docs/blog) on GitHub.
```

Comment:
This is the correct location

```suggestion
 For more information about how to create your own LINQ provider, see [LINQ: Building an IQueryable Provider](https://docs.microsoft.com/archive/blogs/mattwar/linq-building-an-iqueryable-provider-part-i).
```

Reply:
Thanks @mairaw! And the contents of that document seem to be exactly the same as in the [github blog part 1](https://github.com/mattwar/iqtoolkit/blob/master/docs/blog/building-part-01.md). Although the difference is that the MS Docs page does not have pretty markup. Should we do something about that?

Reply:
Not sure if we have permissions to that DevOps repo. But they might give us permissions for us to fix some of these. But they certainly won't prioritize fixing the markup.